README.md

```diff
@@ -1,2 +1,2 @@
-# Java_Refresh
-Learning Java
+# shoprescuespa_automation
+This is a selenium automation project for shoprescuespa.com
```

Comment:
Hi Pooja,

Can you please add more description about the project and technologies used under this?

Reply:
Done. Thanks.